lib/metadataTypes/definitions/Role.definition.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
         'SYS_DEF_SECURITYADMIN',
         'SYS_DEF_VIEWER',
     ],
+    documentInOneFile: true,
```

Comment:
I don't 100% agree with this approach. I actually wanna see document removed from deploy or retrieve. It should be its own thing. Then at CLI level it can be called after if need be on either. It will make the retrieve and deploy more "componentized", not to mention it would probably avoid the need for this modification since it would just take whatever is in the retrieve dir (which should be updated by whatever came from deploy)

Reply:
true - assuming the auto-documentation is a problem. 
on the other hand we used to see updates to our retrieve folder as a feature. and docs are created based on the retrieve folder. by disabling this we lead to difference between whats in retrieve and whats in docs.

it's 2 lines of code that can help us move forward with a fix of existing functionality instead of changing it.


I haven't seen a ticket nor a discussion prior to this bugfix about this, buddy. 
If we want to reduce whats done to docs/retrieve, we can add a new ticket for that and handle it as a feature change.

**Disputing a bugfix over a functional discussion seems wrong though.**